projectroles/tests/test_commands.py

```diff
@@ -39,6 +39,12 @@
 
 EXAMPLE_APP_NAME = 'example_project_app'
 CLEAN_LOG_PREFIX = 'INFO:projectroles.management.commands.cleanappsettings:'
+CLEAN_LOG_DELETE_GHOST = (
+    'Deleting "settings.example_project_app.ghost" from project'
+)
+CLEAN_LOG_DELETE_GHOST_END = 'definition not found'
+CLEAN_LOG_DELETE_INT_SETTING = 'Deleting "settings.example_project_app.project_user_bool_setting" from project'
+CLEAN_LOG_DELETE_INT_SETTING_END = 'does not match allowed types: [\'PROJECT\']'
```

Comment:
Sorry, I was unclear in my earlier point. Instead of redefining these constants here, we should simply import the constants used in cleanappsettings.py and compare our test output to those, with `{placeholders}` in those constants replaced by the appropriate strings using `format()`.

This way, if we decide to make a change to the wording of these log messages, we don't have to also update strings in the test cases.